src/lib/y2network/wicked/connection_config_readers/base.rb

```diff
@@ -70,6 +74,33 @@ def connection_config
 
       private
 
+        def find_bootproto
+          bootproto = BootProtocol.from_name(file.bootproto.to_s)
+          return bootproto if bootproto
+
+          fallback = file.ipaddrs.empty? ? BootProtocol::DHCP : BootProtocol::STATIC
+          issue_location = "file:#{file.path}:BOOTPROTO"
```

Comment:
I am a bit confused, why is here some string and not instance of Y2Issues::Location? I expect something like `issue_location = Y2Issues::Location.new("file", file.path, "BOOTPROTO")`

Reply:
The Y2Issues API supports a "string" that [gets parsed](https://github.com/yast/yast-yast2/blob/master/library/general/src/lib/y2issues/issue.rb#L55). However, I have noticed that if you pass the location directly, it will not work (I will fix it).

Reply:
Here is the PR: https://github.com/yast/yast-yast2/pull/1157.